Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Last-Update: 2012-02-15
Forwarded: https://bugzilla.gnome.org/show_bug.cgi?id=670002
Applied-Upstream: http://svn.gnucash.org/trac/changeset/22025, 2.5.0
Description: convert comments to POD
 POD is helpful not only to view command's documentation but also
 allows to generate man page.


--- a/src/quotes/gnc-fq-check.in
+++ b/src/quotes/gnc-fq-check.in
@@ -26,17 +26,30 @@
 use English;
 use FileHandle;
 
-# Input: <none>
-#
-# Output (on standard output, one output form per input line):
-#
-# A list of quote sources supported by Finance::Quote, or the single
-# term missing-lib if finance quote could not be executed.
-#
-# Exit status
-#
-# 0 - success
-# non-zero - failure
+=head1 NAME
+
+gnc-fq-check  -  check for the presence of Finance::Quote
+                 From gnc-fq-helper
+
+=head1 SYNOPSIS
+
+gnc-fq-check
+
+=head1 DESCRIPTION
+
+Input: <none>
+
+Output (on standard output, one output form per input line):
+
+A list of quote sources supported by Finance::Quote, or the single
+term missing-lib if finance quote could not be executed.
+
+Exit status
+
+0 - success
+non-zero - failure
+
+=cut
 
 sub check_modules {
   my @modules = qw(Finance::Quote LWP HTML::Parser HTML::TableExtract Crypt::SSLeay Date::Manip);
